src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -1138,46 +1245,57 @@ void EnTa_Update(Actor* thisx, PlayState* play) {
     CollisionCheck_SetOC(play, &play->colChkCtx, &this->collider.base);
     Actor_MoveForward(&this->actor);
     Actor_UpdateBgCheckInfo(play, &this->actor, 0.0f, 0.0f, 0.0f, UPDBGCHECKINFO_FLAG_2);
-    this->unk_260(this);
+    this->animFunc(this);
     this->actionFunc(this, play);
 
-    if (!(this->unk_2E0 & 4)) {
-        this->unk_2B0(this);
+    if (!(this->stateFlags & TALON_STATE_FLAG_SUPPRESS_BLINK)) {
+        this->blinkFunc(this);
     }
 
-    if (this->unk_2E0 & 1) {
-        Actor_TrackPlayer(play, &this->actor, &this->unk_2D4, &this->unk_2DA, this->actor.focus.pos);
+    if (this->stateFlags & TALON_STATE_FLAG_TRACKING_PLAYER) {
+        Actor_TrackPlayer(play, &this->actor, &this->headRotStep, &this->torsoRot, this->actor.focus.pos);
```

Comment:
why is one "step" and not the other?
(I think `headRotStep` can just be `headRot`)

Reply:
I thought I renamed both fields to include `step`... apparently not